config.cfg

```diff
@@ -0,0 +1,19 @@
+
+# Configuation file
+[app]
+DEBUG = True
+# This key will be regenerated and hidden at the end of this project
+SECRET_KEY = A0Zr98j/3yX R~XHH!jmN]LWX/,?RT
+SQLALCHEMY_DATABASE_URI = postgres://localhost/lemur
+
+[login_manager]
+SESSION_PROTECTION = strong
+LOGIN_VIEW = login
+
+[logger]
+FILEPATH = error_log/errors.log
+LEVEL = WARNING
+
+[class_time_info]
+CLASS_TIME = 16fall
+CURRENT_TIME = 16fall
```

Comment:
In the current app the admin can change this config file to change the options they have when they create a class. The reality is that the professors only create/use labs for a single semester. So I suppose them to change the values in this file once every semester.. I don't think it's very necessary to create a single web page which is only used to manage semesters....